synedrion/benches/pow.rs

```diff
@@ -0,0 +1,125 @@
+use criterion::{black_box, criterion_group, criterion_main, BatchSize, Criterion};
```

Comment:
I'm not sure if this benchmark needs to be in this repo. It worked as an investigation tool, but doesn't really have anything to do with the protocol itself.

Reply:
Yeah I agree. I struggle to make up my mind on this. Like, yesterday I wrote a benchmark to check if multi-exponentiation would be faster when there is just one exponentiation (i.e. given `x * y^k`, is it faster to do  `x.mul(y.pow(k))` or `multiexp( (x, 1), (y, k) )`?). Totally useless as a long-term benchmark but useful to get an answer; so I just toss the code I wrote in the trash? I guess yes? Or possibly move to a PR against `crypto-bigint`?

Reply:
Ended up removing this, at least for now.